coriolis/osmorphing/base.py

```diff
@@ -96,6 +100,38 @@ def get_packages(self):
 
         return add, remove
 
+    def run_user_script(self, user_script):
+        if len(user_script) == 0:
+            return
+
+        script_path = "/tmp/coriolis_user_script"
```

Comment:
NIT: I'd still use a `tempfile` for safety

For posterity, I'd also consider putting the script in some non-tmp dir. (the temp dir of the worker is bind-mounted during the `chroot` operations so it won't be preserved on the migrated VM)

Reply:
The user has the script, and it will also be a permanent part of the migration task_info. Considering you already have it in 2 places, does it make sense to have it in a 3rd?

Reply:
I need to clarify this a bit more. This is not meant to be copied inside the migrated VM. This is meant to be copied to the worker itself, and run with one parameter: the root_dir of the migrated instance. From there on out, the script can decide what it needs to do to the offline migrated VM.